front/app/component-library/components/Checkbox/index.tsx

```diff
@@ -27,6 +27,7 @@ const StyledCheckbox = styled.div<{
   checkedColor?: Color;
   size: string;
   disabled: boolean;
+  usePrimaryBorder?: boolean;
```

Comment:
```suggestion
  usePrimaryBorder: boolean;
```
We can be stricter by setting a default value of the prop so it's always a boolean. Minor but I like to narrow things and keep code simpler whenever possible.